packages/lib/src/vis/rgb/RgbVis.tsx

```diff
@@ -78,7 +78,7 @@ function RgbVis(props: Props) {
         }}
       >
         <PanZoomMesh />
-        <VisMesh>
+        <VisMesh scale={[1, -1, 1]}>
```

Comment:
no ternary here ?

Reply:
Unneeded as we force the flip of Y is set to `true` above (part of unchanged code)